projects/observability/src/pages/apis/endpoints/endpoint-list.component.ts

```diff
@@ -0,0 +1,13 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { endpointListDashboard } from './endpoint-list.dashboard';
+
+@Component({
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  template: `
+    <div class="vertical-flex-layout">
+      <ht-page-header></ht-page-header>
+      <ht-navigable-dashboard navLocation="${endpointListDashboard.location}"></ht-navigable-dashboard>
+    </div>
+  `
+})
+export class EndpointListComponent {}
```

Comment:
Have we have started calling apis as endpoints in the code as well? Would that cause confusion with existing components for API like `ApiDetailBreadcrumbResolver`?

Reply:
It's a bit mixed in code. We should get it all consistent, but figured if we're adding new stuff to make it accurate. Another change that should happen is flattening the structure of pages (removing the `/apis` parent dir)